app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt

```diff
@@ -129,22 +153,18 @@ class NewConversationViewModel @Inject constructor(
         }
     }
 
-    private fun observeGroupNameChanges() {
-        viewModelScope.launch {
-            newGroupNameTextState.textAsFlow()
-                .dropWhile { it.isEmpty() } // ignore first empty value to not show the error before the user typed anything
```

Comment:
the `dropWhile { it.isEmpty() }` is not needed anymore since we will observe this on the UI scope not the viewModel